### PR TITLE
KOGITO-8728: Enable the Problems tab for Dashboards

### PR DESCRIPTION
**Pull request by caponetto**
https://issues.redhat.com/browse/KOGITO-8728

Also in this PR:
- Reorganize the `EditorPage`
- Create components to instantiate file-specific APIs only when necessary
- Re-enable warnings when `operationId` is missing from service functions

https://user-images.githubusercontent.com/638737/226328540-9c0ffcc8-b738-4905-9cdb-66f6fab03dd7.mp4

### CODE REVIEW

**Review by jesuino — APPROVED** [review]
LGTM!

File: packages/serverless-logic-web-tools/src/editor/WebToolsEmbeddedEditor.tsx [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+      setReady(true);
+    }
+  }, [channelComponent, isReady, file]);
+
+  return (
+    <>
+      <EmbeddedEditor
+        /* FIXME: By providing a different `key` everytime, we avoid calling `setContent` twice on the same Editor.
+         * This is by design, and after setContent supports multiple calls on the same instance, we can remove that.
+         */
```
Comment (tomasdavidorg) [inline packages/serverless-logic-web-tools/src/editor/WebToolsEmbeddedEditor.tsx]: I think we should link this to some jira.

**Review by tomasdavidorg — APPROVED** [review]
Looks good to me 🙂. 
Just 2 points:
- There is no Jira link to FIXME. See the comment.
- The Dashbuilder examples `Dashbuilder Kitchensink` and `Serverless Workflow Report` have the warnings in the panel. Is it intended? I think we should fix those examples.

File: packages/serverless-logic-web-tools/src/editor/WebToolsEmbeddedEditor.tsx (hunk 1 above)
Comment (caponetto) [reply packages/serverless-logic-web-tools/src/editor/WebToolsEmbeddedEditor.tsx]: Create this one https://issues.redhat.com/browse/KOGITO-8892 and linked it to the code.

**Comment by caponetto** [comment]
Hi @tomasdavidorg,

Thanks, I created these two tickets:
- https://issues.redhat.com/browse/KOGITO-8891 (cc @jesuino)
- https://issues.redhat.com/browse/KOGITO-8892

**Comment by tomasdavidorg** [comment]
@caponetto Thanks.

**Review by paulovmr — APPROVED** [review]

**Pull request merged**